docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+
+## ClickHouse connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Delta Lake connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Rename `s3.exclusive-create` config to `delta.s3.transaction-log-conditional-writes.enabled`. ({issue}`27372`)
+* Fix incorrect results due to incorrect pushdown of `IS NOT DISTINCT FROM`. ({issue}`27213`)
+* Fix failure when writing to tables written by Databricks 17.3. ({issue}`27100`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+* Prevent log writer from failing with FileAlreadyExistsException when there are network failures during write. 
+  writing to s3 now requires permissions for PutObjectTagging and GetObjectTagging operations (breaking change). ({issue}`27388`)
+* Fix potential failure when reading [cloned tables](https://docs.databricks.com/aws/en/delta/clone). ({issue}`27098`)
+
+## Druid connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## DuckDB connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Elasticsearch connector
+
+## Exasol connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Remove `s3.exclusive-create` config. ({issue}`27372`)
+* Fix updating table statistics when running INSERT queries with OVERWRITE behaviour. ({issue}`26517`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+
+## Hudi connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+* Fix failure when listing empty directories on HDFS. ({issue}`26897`)
```

Comment:
Users don't know when that listing happens. Describe it in terms of concepts users can relate to.

Reply:
How about "Fix failure when querying HDFS that includes empty directories."

Reply:
applied above, FYI